examples/ffva/src/usb/adaptive_rate_callback.c

```diff
@@ -83,22 +50,68 @@ uint32_t sum_array(uint32_t * array_to_sum, uint32_t array_length)
     return acc;
 }
 
-void reset_state()
+
+float_s32_t float_div(float_s32_t dividend, float_s32_t divisor)
```

Comment:
this is really clever - is there a reason you're not using float_s32_div? I see that you've gone so far as to comment it out below?

Reply:
The float_s32_div works on signed numbers. We can get a couple of extra bits of precision by doing unsigned division so decided to write my own.